scripts/genomes/parse_genome_assemblies.py

```diff
@@ -0,0 +1,208 @@
+"""Parse genome assemblies from NCBI by their level of completion.
+
+This script fetches current and historical Assembly Reports from NCBI, and
+extracts metadata on assemblies that are relevant for Single Cell Portal.
+
+Only assemblies that are chromosome-level or above are included.
+
+For definitions of assembly-related terms and other domain background, see:
+https://www.ncbi.nlm.nih.gov/assembly/help/#glossary
+"""
+
+import argparse
+from functools import cmp_to_key
+import os
+import urllib.request as request
+
+parser = argparse.ArgumentParser(
+    description=__doc__, # Use docstring at top of file for --help summary
+    formatter_class=argparse.RawDescriptionHelpFormatter)
+parser.add_argument('--output_dir',
+                    help='Directory to sent output data to.  Default: output/',
+                    default='output/')
+args = parser.parse_args()
+output_dir = args.output_dir
+
+if os.path.exists(output_dir) is False:
+    os.mkdir(output_dir)
+
+# Priority species to support in Single Cell Portal
+species_list = [
+    ['Homo sapiens', 'human', '9606'],
+    ['Mus musculus', 'mouse', '10090'],
+    ['Danio rerio', 'zebrafish', '7955'],
+    ['Gallus gallus', 'chicken', '9031'],
+    ['Rattus norvegicus', 'rat', '10116'],
+    ['Macaca fascicularis', 'macaque', '9541'],
+    ['Bos taurus', 'cow', '9913'],
+    ['Sus scrofa', 'pig', '9823'],
+    ['Canis lupus familiaris', 'dog', '9615'],
+    ['Felis catus', 'cat', '9685']
+]
+
+# Sorted list of species names
```

Comment:
Does the result of the code below generate a _sorted_ list?

Reply:
Yes; saying "sorted list" was actually redundant of me.  I've revised this comment to "One-dimensional list of species names" in my current topic branch `ew-genome-annotations`.

I will also add a comment on what sorting is used here in that other branch.  In brief, they are sorted by "priority", which is the order we'd ideally like to show the species in the UI when listed.